Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["contracts/bindings", "types", "sdk"]
+members = ["contracts/bindings", "types", "rust-sdk"]
```

Comment:
oh, this rename might also break some stuff at the other repos, I'll take a look